controller_manager:
  ros__parameters:
    update_rate: 100
    use_sim_time: true

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster
    simple_position_controller:
      type: position_controllers/JointGroupPositionController

joint_state_broadcaster:
  ros__parameters:
    publish_rate: 50

simple_position_controller:
  ros__parameters:
    joints:
      - shoulder_pan_joint
      - shoulder_lift_joint
      - elbow_joint
      - wrist_1_joint
      - wrist_2_joint
      - wrist_3_joint
    state_publish_rate: 25
    command_publish_rate: 25
    constraints:
      goal_time: 0.5
      stopped_velocity_tolerance: 0.01
    gains:
      shoulder_pan_joint:
        p: 100.0
        d: 0.1
        i: 1.0
        i_clamp: 0.5
      shoulder_lift_joint:
        p: 100.0
        d: 0.1
        i: 1.0
        i_clamp: 0.5
      elbow_joint:
        p: 100.0
        d: 0.1
        i: 1.0
        i_clamp: 0.5
      wrist_1_joint:
        p: 100.0
        d: 0.1
        i: 1.0
        i_clamp: 0.5
      wrist_2_joint:
        p: 100.0
        d: 0.1
        i: 1.0
        i_clamp: 0.5
      wrist_3_joint:
        p: 100.0
        d: 0.1
        i: 1.0
        i_clamp: 0.5
